### DISCUSSION TITLE
Isolated mode with only the Editor + icons?

### DISCUSSION THREAD

**Discussion by dmccallie** [pd4d10/hashmd#272] [Q&A]
Hi all,

Is there any way to configure so that only the Editor (not the preview) window is shown, while also showing the icons in the toolbar?

So far, I can only see the toolbar icons when in "split" mode, and with the screen set very wide.

I might have missed a config option?

--david

**Comment by CodeGetters** [comment]
I didn't find the relevant configuration in the warehouse, so I came up with another solution: set the preview style to none directly.👀
